index.css

```diff
@@ -11,16 +11,19 @@ main {
   width: 375px;
   height: 5410px;
   background: #fff;
+  margin-left: 30px;
+  display: grid;
 }
 
-.full-wrapper {
+/* .full-wrapper {
   position: absolute;
   width: 375px;
   height: 718px;
   left: 1px;
   background-color: #fffffffa;
   align-items: center;
-}
+  background-color: #396DF2;
+} */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.